[2023-06-04T08:02:08.100+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: most_popular_movies.load_movies manual__2023-06-04T08:02:02.881908+00:00 [queued]>
[2023-06-04T08:02:08.106+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: most_popular_movies.load_movies manual__2023-06-04T08:02:02.881908+00:00 [queued]>
[2023-06-04T08:02:08.107+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-04T08:02:08.117+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): load_movies> on 2023-06-04 08:02:02.881908+00:00
[2023-06-04T08:02:08.121+0000] {standard_task_runner.py:57} INFO - Started process 146 to run task
[2023-06-04T08:02:08.124+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'most_popular_movies', 'load_movies', 'manual__2023-06-04T08:02:02.881908+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/most_popular_movies.py', '--cfg-path', '/tmp/tmp7bmhaftm']
[2023-06-04T08:02:08.124+0000] {standard_task_runner.py:85} INFO - Job 49: Subtask load_movies
[2023-06-04T08:02:08.156+0000] {task_command.py:410} INFO - Running <TaskInstance: most_popular_movies.load_movies manual__2023-06-04T08:02:02.881908+00:00 [running]> on host 7633a4fabdb3
[2023-06-04T08:02:08.344+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='most_popular_movies' AIRFLOW_CTX_TASK_ID='load_movies' AIRFLOW_CTX_EXECUTION_DATE='2023-06-04T08:02:02.881908+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-04T08:02:02.881908+00:00'
[2023-06-04T08:02:08.351+0000] {logging_mixin.py:149} INFO - Fetching Dataset...
[2023-06-04T08:02:08.908+0000] {logging_mixin.py:149} INFO - Done
[2023-06-04T08:02:08.908+0000] {logging_mixin.py:149} INFO - https://bigquery.googleapis.com/bigquery/v2/projects/imdb-388708/datasets/imdb
[2023-06-04T08:02:08.909+0000] {logging_mixin.py:149} INFO - 
Fetching Table...
[2023-06-04T08:02:09.181+0000] {logging_mixin.py:149} INFO - Done
[2023-06-04T08:02:09.181+0000] {logging_mixin.py:149} INFO - https://bigquery.googleapis.com/bigquery/v2/projects/imdb-388708/datasets/imdb/tables/most_popular_movies
[2023-06-04T08:02:09.182+0000] {logging_mixin.py:149} INFO - 
Loading data into BigQuery...
[2023-06-04T08:02:09.182+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/decorators/base.py", line 220, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/most_popular_movies.py", line 36, in load_movies
    _load_bigQuery_most_popular_movies(movies_df)
  File "/opt/airflow/dags/helper/web_scrape.py", line 202, in _load_bigQuery_most_popular_movies
    movies_df, table_id, job_config=job_config
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/client.py", line 2635, in load_table_from_dataframe
    dataframe, job_config.schema
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/_pandas_helpers.py", line 400, in dataframe_to_bq_schema
    bq_schema_unused
ValueError: bq_schema contains fields not present in dataframe: {'movie_id'}
[2023-06-04T08:02:09.189+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=most_popular_movies, task_id=load_movies, execution_date=20230604T080202, start_date=20230604T080208, end_date=20230604T080209
[2023-06-04T08:02:09.197+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 49 for task load_movies (bq_schema contains fields not present in dataframe: {'movie_id'}; 146)
[2023-06-04T08:02:09.219+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-04T08:02:09.231+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
